modules/social_features/social_group/modules/social_group_welcome_message/social_group_welcome_message.module

```diff
@@ -168,7 +168,14 @@ function social_group_welcome_message_group_content_insert(GroupContentInterface
     }
 
     // Merge all managers of the group to recipients array.
-    $recipients = array_merge($g_admins_users, $g_managers_users, [$group->getOwner()]);
+    $recipients = array_merge($g_admins_users, $g_managers_users);
+
+    // Check what groups the group owner is in, before adding this person
+    // to the list of recipients.
+    $owner_groups = \Drupal::service('social_group.helper_service')->getAllGroupsForUser($group->getOwnerId());
```

Comment:
@sjoerdvandervis I do feel that we should remove "Group owner" at all from this equation, we should only include group managers. By default, in Open Social, the group creator is assigned with "group manager" role. These settings are present in each group type, ex: /admin/group/types/manage/closed_group. This will also solve the situation where a group creator leaves.

Reply:
Thanks for this, we indeed don't need this check anymore and we don't want the group owner in the list of recipients, as the group owner already has the group manager role indeed. So we can just remove the group owner from the list of recipients.